packages/@atjson/offset-annotations/src/annotations/accordion-group.ts

```diff
@@ -0,0 +1,23 @@
+import { BlockAnnotation } from "@atjson/document";
+/**
+ * An Accordion contains a collection of accordion items that
+ * are arranged vertically.
+ * Each accordion item contains a header and a panel
+ * The header contains the title of the content in the panel
+ * The panel could contain text, pictures, embeds, videos
+ *
```

Comment:
This comment seems to be incorrectly attributed to this block. Could you make a new comment or remove this?

Reply:
changed, pls review